### DISCUSSION TITLE
Issue loading .pcd files with io.load()

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#488] [Q&A] [answered]
Hi!
Am having troubles loading a .pcd point cloud file using io.load():

![pcd_edit](https://user-images.githubusercontent.com/8246894/138031765-41816202-98e7-47f1-9917-ba0401c17983.png)

I know the pcd file works becasue I can open it file with other programs and open3d. Any ideas on why I'm getting this response? Do I need to preprocess in some fashion?

Cheers 
J

**Comment by marcomusy** [comment, marked as answer]
My first guess is that your file is compressed (?), at present only ascii (non binary) pcd format is supported.
If you can read the file in some other way then you just need to create a `vedo.Points(coords)` object.

**Reply by jkissi** [reply]
Ok. the program I use to save doesn't give me any options when I save as pcd so I will assume you are correct. It's not critical for me anyway. Any idea when binary .pcd saves will be possible?
Once again, cheers
J

**Reply by marcomusy** [reply]
Hi, sorry for the late reply, at present we don't have the possibility to save pcd as binary.. I'm not expert on pcd format so it's difficult to say if that will ever be possible...

**Reply by jkissi** [reply]
Ok, cheers for the info
